packages/aila/src/protocol/schema.ts

```diff
@@ -1,79 +1,75 @@
+import dedent from "dedent";
 import z from "zod";
 import { zodToJsonSchema } from "zod-to-json-schema";
 
+import { minMaxText } from "./schemaHelpers";
+
+// ********** BASED_ON **********
+export const BASED_ON_DESCRIPTIONS = {
+  id: dedent`The database ID of an existing lesson plan that this
+    lesson plan is based upon.
+    It should be the ID of the lesson plan that was used to inform
+    the content of this lesson plan.
+    It should not be the number that the user has selected when
+    choosing which lesson on which to base their new lesson.`,
+  title: "The human-readable title of the lesson.",
+  schema: dedent`A reference to a lesson plan that this lesson is
+    based on. This value should only be set if the user has explicitly
+    chosen to base their lesson on an existing lesson plan by selecting
+    one from a selection of options, otherwise this should be blank.`,
+} as const as { id: string; title: string; schema: string };
```

Comment:
I'm not familiar with `as const as`. Maybe you're thinking of `satisfies`?

Reply:
I also tried satisfies and it gave me type complaints. Similarly - not familiar with const as!

Reply:
Does `as { id: string; title: string; schema: string }` work as expected?